dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7785,7 +7785,7 @@ public String toString() {
 
   public static final PropertyKey DORA_CLIENT_READ_LOCATION_POLICY_ENABLED =
       booleanBuilder(Name.DORA_CLIENT_READ_LOCATION_POLICY_ENABLED)
-          .setDefaultValue(false)
+          .setDefaultValue(true)
```

Comment:
i actually think we should separate this line of change to a separate PR like: "Enable Dora architecture by default" That's probably including removing key `WORKER_BLOCK_STORE_TYPE`

Reply:
Per discussion, this switch change itself cannot be achieved without the UT changes (because the UTs simply can't pass). So I give up and let's do it here.

Reply:
should we rename this property? 
I feel this is the main switch of dora so maybe we should rename it to be alluxio.dora.enabled or somethign?

Reply:
absolutely

Reply:
IMO it is fine whether we rename in a separate PR or do it here. If there's not much compatibility issue renaming it, then let's do it here.